module_utils/foreman_helper.py

```diff
@@ -244,6 +244,9 @@ def _generate_resource_params(self, resource, action, resource_id=None, params=N
                 resource_params[hash_entry_name] = params
             else:
                 resource_params.update(params)
+        if action == 'create':
```

Comment:
can you add a comment that his is a special case for `organization` which wants `name` in both places?